tools/degas.bash

```diff
@@ -55,7 +63,7 @@ for file; do
         if $interactive; then
             read -r -p "Convert $file to $ext? [y/n] " confirmation
             if [[ $confirmation != [Yy] ]]; then
-                continue
+                return
```

Comment:
```suggestion
                continue
```

Reply:
This was accidentally left over from a test of refactoring this into a function.  Should be continue.